book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -125,21 +125,21 @@ $ git status -sb
 ## master
 ----
 
-The `git merge --abort` option tries to revert back to your state before you ran the merge.
-The only cases where it may not be able to do this perfectly would be if you had unstashed, uncommitted changes in your working directory when you ran it, otherwise it should work fine.
+Ang `git merge --abort` na opsyon na susubukang ibalik muli sa estado bago magpatakbo ng merge.
+Ang tanging mga kaso na kung saan ay hindi makagawa ng perpektong kung ito ay hindi naitago, ang mga hindi nai-commit na mga pagbabago sa iyong tinatrabahong direktoryo kapag pinatakbo mo ito, kung hindi man ito ay dapat gumagana.
```

Comment:
- `hindi makagawa ng perpektong` to `hindi perpektong makagawa`
- `kung hindi man ito ay dapat gumagana` to `kung hindi man ay dapat itong gumana ng maayos`